subiquitycore/async_helpers.py

```diff
@@ -45,9 +45,9 @@ def schedule_task(coro, propagate_errors=True):
 background_tasks = set()
 
 
-def run_bg_task(coro, *args, **kwargs) -> None:
```

Comment:
I think that would be good yes 👍

Reply:
Actually, we can't really do:

```python
def run_bg_task(coro, *, name=None, context=None):
[...]
    asyncio.create_task(coro, name=name, context=context)
```

because context was added in Python 3.11 (which is newer than what we use in production code)

Reply:
Well OK I guess it should just be name=None then :-)